pkg/certificate/mock_certificate_test.go

```diff
@@ -0,0 +1,57 @@
+package certificate
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	tassert "github.com/stretchr/testify/assert"
+)
+
+// Silliest thing - "go test -cover" would report the mocks as not tested and show very low test coverage.
```

Comment:
Instead of adding tests for mocks to give the illusion of improved code coverage, can we just strip them from the coverage report?
https://stackoverflow.com/questions/50065448/how-to-ignore-generated-files-from-go-test-coverage
```
go test . -coverprofile cover.out.tmp
cat cover.out.tmp | grep -v "_generated.go" > cover.out
tool cover -func cover.out
```

Reply:
I like that. Let me hack on it.